miqa/core/management/commands/populate_demo.py

```diff
@@ -0,0 +1,39 @@
+from allauth.account.admin import EmailAddress
+from django.contrib.auth.models import User
+import djclick as click
+from guardian.shortcuts import assign_perm
+
+from miqa.core.models import Project
+from miqa.core.tasks import import_data
+
+
+@click.command()
+def command():
+    # Delete extra projects in the system that users may have created
```

Comment:
This should check the value of `DEMO_MODE` before doing anything else

Reply:
Changed in [5cb08e8](https://github.com/OpenImaging/miqa/pull/449/commits/5cb08e840ddb8115d0c096db10c908b61b2202ea)